server/util.go

```diff
@@ -354,3 +356,17 @@ func appendFormatFloat(in []byte, fVal float64, prec, bitSize int) []byte {
 	}
 	return out
 }
+
+// CorsHandler add Cors Header if `cors` config is setted
+type CorsHandler struct {
+	handler http.Handler
+	cfg     *config.Config
+}
+
+func (h CorsHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+	if h.cfg.Cors {
+		w.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
@xuechunL do we need set more special address?

if so maybe we can change configuration from enable/disable to a host string? and set that value via ansible?